frontend/apps/core-app/src/components/RecordTable/__tests__/GlobalTableFilter.test.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import { render } from '../../../testUtils/render';
+import { GlobalTableFilter } from '../GlobalTableFilter';
+
+it('should match the snapshot', () => {
```

Comment:
This probably needs some tests to check that changing the input calls the correct handlers